### PR TITLE
Update tornado to 6 for Python 3

### PR DESCRIPTION
**Pull request by Dith3r**
Update tornado to lastest version for Python 3, swtich ioloop use uvloop and json to ujson.

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
@Dith3r `tornado: No matching distribution found for tornado==6.0.2`

**Comment by ifduyue** [comment]
[Tornado 6.0.2 exists](https://pypi.org/project/tornado/6.0.2/), but it only supports Python 3.

File: frameworks/Python/tornado/commons.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import json
+import ujson as json
```
Comment (ifduyue) [inline frameworks/Python/tornado/commons.py]: Why ujson? It seems ujson is dead. Lastest version v1.3.5 was released 3 years ago.

Comment (Dith3r) [reply frameworks/Python/tornado/commons.py]: > frameworks/Python/aiohttp/requirements.txt:ujson==1.35
> frameworks/Python/blacksheep/requirements.txt:ujson==1.35
> frameworks/Python/bottle/requirements.txt:ujson==1.35
> frameworks/Python/django/requirements.txt:ujson==1.35
> frameworks/Python/django/requirements_py3.txt:ujson==1.35
> frameworks/Python/fastapi/requirements.txt:ujson==1.35
> frameworks/Python/japronto/requirements.txt:ujson==1.35
> frameworks/Python/responder/requirements.txt:ujson==1.35
> frameworks/Python/starlette/requirements.txt:ujson==1.35
> frameworks/Python/uvicorn/requirements.txt:ujson==1.35
> frameworks/Python/uwsgi/requirements.txt:ujson==1.35
> frameworks/Python/vibora/requirements.txt:ujson==1.35
> frameworks/Python/wsgi/requirements.txt:ujson==1.35
It's used in multiple places.

Comment (ifduyue) [reply frameworks/Python/tornado/commons.py]: Looking at usjon's opening issues and current status I will not consider using usjon in production. Just my two cents.

**Comment by nbrady-techempower** [comment]
Going to close and reopen this; looks like travis missed the commit trigger.

**Comment by nbrady-techempower** [comment]
@Dith3r Need to update the `tornado.dockerfile`

should be:
```
RUN pip install -r /tornado/requirements_py2_mongo.txt
```

**Pull request closed without merging**